### DISCUSSION TITLE
[Android] Notification dot not dissapearing after clear notification tray?

### DISCUSSION THREAD

**Discussion by agreedSkiing** [tutao/tutanota#6530] [Q&A]
Since the 20th October 2023 I have had an issue. A small one nothing major one regarding notifications. The notification dot does not  disappear when I clear my notification tray, that it. This just bugs me and why is that happening 🫤🧐?

I know how to remove it (the notification dot), either by pausing the app and then reactivating it or restarting my phone, oh right almost forgot the third way that sometimes works 😅... If I click on a notification to clear it notification and re-authenticate/unlock the app if locked or just open it if its running in the background and do that to every single one of the notifications it.

One thing to note is that i use my notification as reminders for the day, so I set them up around 5 minutes before I wake up so that they trigger in my do not disturbed window and the events are only a minute long since I just want the notification.

Is this a common not reported issue?

*System info*
~~~
Client: android
Type: FREE
Tutanota version: 3.121.4
Timestamp (UTC): Fri, 09 Feb 2024 17:20:49 GMT
User agent:
Mozilla/5.0 (Linux; Android 13; Pixel 4a Build/TQ3A.230805.001.S1; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/121.0.6167.143 Mobile Safari/537.36
~~~
![Screenshot_20240209-182019](https://github.com/tutao/tutanota/assets/93997033/b745221c-d62c-4e5a-94fd-4e2c557d28c6)
![Screenshot_20240209-182029](https://github.com/tutao/tutanota/assets/93997033/e0ca51f2-dae9-41fb-9791-a499e41f755c)

P.S. I am now experiencing something new where I do not get a dot at all...

**Comment by charlag** [comment]
Hi, thanks for reporting, I don't remember hearing any issues about this so far but we should probably check
